.github/workflows/cd.yaml

```diff
@@ -0,0 +1,42 @@
+name: Build and deploy
+
+on:
+  workflow_run:
+    workflows: ["Check and test"]
+    branches: [main]
+    types:
+      - completed
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    needs: [lint, test]
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16.3
+
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Build
+        env:
+          GITHUB_ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+        run: |
+          make image
+#      - name: Docker Build and Push
```

Comment:
Should we just remove these comments, or will they be uncommented at some point?